vexim/adminfail.php

```diff
@@ -35,6 +35,8 @@
               . '<td align="center">'
               . '<a href="adminfaildelete.php?user_id='
               . $row['user_id']
+              . '&localpart='
```

Comment:
The right way would be to query database and get it from there just before asking.
Anyway, I just want you to escape the `&` character if you'll be pushing any other changes to this branch. If not, this can stay as is.